ledger-core/v2/virtual/execute/execute.go

```diff
@@ -109,29 +113,37 @@ func (s *SMExecute) stepWaitObjectReady(ctx smachine.ExecutionContext) smachine.
 		isConstructor = true
 		isOrdered = true
 		s.execution.Object = reference.NewGlobalSelf(s.Payload.CallOutgoing)
-		objectSharedState = objectCatalog.GetOrCreate(ctx, s.execution.Object)
 
-	case payload.CTInboundAPICall, payload.CTOutboundAPICall, payload.CTMethod, payload.CTNotifyCall:
+	case payload.CTMethod:
+		isConstructor = false
+		isOrdered = true
+
+	case payload.CTInboundAPICall, payload.CTOutboundAPICall, payload.CTNotifyCall:
```

Comment:
it can be just 'default'

Reply:
It's listing of all call types that we need to be able to process in the future.
It can be default, but it also needs to be a reminder for programmer.

Reply:
ok